### ISSUE TITLE
Unhelpful "check the logs" error message in certain scenarios

### ISSUE THREAD

**Issue by ceball** [jpmorganchase/nbcelltests#154]
![Screenshot from 2020-05-28 12-45-25](https://user-images.githubusercontent.com/1929/83137481-24943580-a0e1-11ea-985b-052c83a85a3f.png)

This happens if there's an "unhandled exception" in nbcelltests. nbcelltests should probably be better about capturing and reporting such errors, so the underlying fix might be in celltests.

Cases I can think of off the top of my head:
  * syntax error in notebook or tests (#101)
  * test class generation failure (e.g. test of non-code cell)

**Comment by vidartf** [comment]
> nbcelltests should probably be better about capturing and reporting such errors

I would agree that this should be the actionable point here. I don't think there is any reason to be restrictive about error reporting here, unless the server has been configured to be restrictive about it (e.g. to prevent leaking file paths giving attackers information about the system).
